textx/model.py

```diff
@@ -459,33 +496,46 @@ def process_match(nt):
         """
         Process subtree for match rules.
         """
+        line, col = parser.pos_to_linecol(nt.position)
         if isinstance(nt, Terminal):
-            return metamodel.convert(nt.value, nt.rule_name)
+            return metamodel.process(nt.value, nt.rule_name,
+                                     filename=parser.file_name,
+                                     line=line, col=col,
+                                     _default_processor=lambda x: x)
```

Comment:
I feel uneasy about `_default_processor` param. If we can figure out when this identity transformation should be used directly in the `process` method it would make these calls cleaner. When identity transformation is used always when there are no other processors two test cases are failing (related to calling processors in the context of abstract rules).

Reply:
My thoughts:
 * We could pass the type of rule. But this would be the same as passing a flag - or the function itself.
 * We could extract the information from the metamodel... I need to investigate how to do this - but it should be easy.
 * We could only **call process in the last stage of model creation only if an object processor is defined**. I try this - so we could discuss it. You are right - this seems maybe better. 

--> selected last version. updated code to be discussed.